streams.yaml

```diff
@@ -14,57 +14,132 @@ proxima.polygon-mumbai.blocks.1_0:
   description: "block headers of polygon mumbai testnet"
   labels:
     encoding: "json"
+proxima.eth-main.blocks-sync.1_0:
```

Comment:
let's keep these stream private by now, they have a meaning for our internal apps only